src/crypto/aggsig.rs

```diff
@@ -76,13 +76,13 @@ impl PublicKey {
 /// An individual signature as part of the aggregate signature scheme.
 ///
 /// This is a wrapper around [`blst::min_sig::Signature`].
-#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
This only really makes sense because BLS signatures happen to be deterministic and unique. Maybe add a "NOTE" comment that these traits should be removed in case we change the underlying signature scheme?